lib/open_api_spex/controller.ex

```diff
@@ -0,0 +1,153 @@
+defmodule OpenApiSpex.Controller do
+  @moduledoc ~S'''
+  Generation of OpenAPI documentation via ExDoc documentation and tags.
+
+  ## Supported OpenAPI fields
+
+  Attribute `operationId` is automatically provided by the implementation
+  and cannot be changed in any way. It is constructed as `Module.Name.function_name`
```

Comment:
Can you make it overridable? I find that `MyAppWeb.ControllerName.function_name` doesn't translate well in all cases.

Reply:
I can, however it will prevent optimisations suggested by @mbuhot in https://github.com/open-api-spex/open_api_spex/pull/99#issuecomment-483489501

Reply:
I'm researching a way to parse specs from `@doc` attributes at compile time, so that it doesn't need to be parsed on each request.